manylinux1-wheel-build/README.md

```diff
@@ -0,0 +1,31 @@
+# Wheel Builder image
+
+This image is a little different than the other images in this repo --
+it's intended to be a quick way to build manylinux wheels with the
+production version of the libraries for debugging and testing
+purposes. This is designed to work as a single docker image that
+doesn't require privlege or multiple images to run.
+
+Like the other images in this repo, it expects that there is a Pillow
+source directory mounted at /Pillow. Unlike the others, it puts the
+output files in /output. This should be mounted as a volume to the host
+to retrieve the finished wheel.
+
+The Makefile has several new commands:
+
+* make wheel: Makes a python 3.8 many linux wheel, and puts it in the
+./out directory.
+* make 36|37|38|39: These are specific commands to make
+the corresponding 3.x version in the ./out directory.
+
+The test target here is mainly to validate the image build, it is
+assumed that the builds.
```

Comment:
This sentence doesn't appear finished?

Reply:
Addressed by https://github.com/python-pillow/docker-images/pull/115/commits/89eccb423f0428aa2762d280fe0cb98a8bdc83a4